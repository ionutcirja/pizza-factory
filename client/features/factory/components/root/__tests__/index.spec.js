import React from 'react';
import { shallow } from 'enzyme';
import Factory from '..';

describe('Factory component', () => {
  describe('render', () => {
    it('should render a sizes query component', () => {
      const wrapper = shallow(<Factory />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
