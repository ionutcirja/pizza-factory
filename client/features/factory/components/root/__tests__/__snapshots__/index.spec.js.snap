// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Factory component render should render a sizes query component 1`] = `
<form>
  <SizesQuery
    Component={[Function]}
    ErrorWrapper={[Function]}
    LoadingWrapper={[Function]}
  />
</form>
`;

exports[`Sizes component render should render a select form field component 1`] = `
<FormField
  name="size"
  options={
    Array [
      "option 1",
      "option 2",
      "option 3",
      "option 4",
    ]
  }
  placeholder="Select pizza size"
  type="select"
/>
`;
